addons/mail/static/src/models/composer/composer.js

```diff
@@ -160,7 +158,7 @@ function factory(dependencies) {
                 Object.assign(postData, {
                     context,
                     subtype_id,
```

Comment:
`subtype_id` is useless if `subtype_xmlid` is properly set (which it always is here), so we can remove it